src/dbcp/extract/eip_infrastructure.py

```diff
@@ -29,44 +58,24 @@ def _downcast_ints(df: pd.DataFrame) -> None:
 
 
 def extract(path: Path) -> Dict[str, pd.DataFrame]:
-    """Read EIP excel database.
+    """Read in EIP CSV files from a provided path to a folder.
 
     Args:
         path (Path): filepath
 
     Returns:
         Dict[str, pd.DataFrame]: output dictionary of dataframes
```

Comment:
I don't think think we need the `path` arg anymore now that it's returned by `cache_gcs_archive_file_locally`. Also we can remove the type hints in the doc string.